journeys/et/updateIP.ts

```diff
@@ -0,0 +1,25 @@
+import {Journey} from "types/journey"
+import {readFileSync, writeFileSync} from "fs";
+import {sep} from "path";
+import {execCommand} from "app/helpers";
+
+async function updateIP() {
+  const ip = (await execCommand("hostname -I | awk '{print $1}'")).stdout.trim();
+  updateRegion(process.env.ENGWALES_DEF_DIR, ip)
+  updateRegion(process.env.SCOTLAND_DEF_DIR, ip)
+}
+
+function updateRegion(regionDir: string, ip: string) {
+  const packageJsonPath = `${regionDir}${sep}package.json`;
+
+  const json = JSON.parse(readFileSync(packageJsonPath).toString())
```

Comment:
should be able to use require here, it'll parse json automatically

Reply:
What do you mean by require?

Reply:
```
  const packageJsonPath = `${regionDir}${sep}package.json`
  const packageJson = require(packageJsonPath)

  packageJson.config.local.et_cos = `http://${ip}:8081`
  writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2))
 ```